citations-tool/tool/src/java/org/sakaiproject/citation/tool/CitationHelperAction.java

```diff
@@ -1121,6 +1125,43 @@ protected void doPostJsonResponse(ParameterParser params,
 		return results;
 	}
 
+	protected Map<String, Object> updateIntroduction(ParameterParser params, SessionState state) {
+		Map<String, Object> results = new HashMap<String, Object>();
+		String message;
+		try {
+			String resourceUuid = params.getString("resourceUuid");
+			String resourceId = this.getContentService().resolveUuid(resourceUuid);
+			String introduction = params.get("addSectionHTML");
+			introduction = getFormattedText().processFormattedText(introduction, new StringBuilder(), true, true);
+			introduction = introduction.replaceAll("'", "&apos;");
+			getContentService().addProperty(resourceId, ResourceProperties.PROP_INTRODUCTION, introduction);
```

Comment:
Sorry I forgot to link this the request.  It's linked to https://github.com/ox-it/wl-kernel/pull/71